client/components/first-view/index.jsx

```diff
@@ -43,7 +43,9 @@ const FirstView = React.createClass( {
 	},
 
 	componentWillUnmount() {
-		this.updateDocumentStylesForHiddenFirstView();
+		process.nextTick( () => {
```

Comment:
Agreed -- think setTimeout would be better in this case, since it is a UI component.